README.md

```diff
@@ -182,10 +210,96 @@ Here the steps:
    * branch            main       -> FETCH_HEAD
    Successfully mreged and updated refs/heads/main.
    ```
+## Contributing through a branch
 
----
+The following explains the steps to [contribute directly to a repository using
+a branch](https://docs.github.com/en/github/getting-started-with-github/quickstart/github-flow).
 
-## Resources
+In a nutshell, here the steps:
+
+1. Clone the repository on your local computer
 
+   :bangbang: **NOTE** This is different from the above as you are cloning the original
+   repository and not your fork.
+   
+   Command:
+   ```shell
+   git clone https://github.com/snowex-hackweek/learning-git
+   ```
+
+1. Create a branch
+
+   Choose a unique name that is not already taken in the repository and helps
+   other collaborators to identify your work. In this case we are adding a branch
+   with name 'feature_xyz'
+   
+   ```shell
+   # The '-b' is indicating a new branch with the name `feature_xyz`
+   # It will also change to that.
+   git co -b feature_xyz
+   ```
+   
+   Sample output:
+   ```shell
+   Switched to a new branch 'feature_xyz'
+   ```
+   
+1. Verify you are on the new branch
+
+    ```shell
+    # Show all branches of the repository, the current active has the `*`
+    git branch
+    ```
+
+   Output:
+    ```shell
+    main
+    * feature_xyz
+    ```
+
+1. Make changes within the new branch
+
+   With this practice repository, you have to create a folder within
+   **`contributions`** folder with your github username as the folder name. Use
+   a new markdwon file (`.md`) or text file (`.txt`) file to practice git with.
+   No other file types or folders are allowed outside your designated folder
+   with this practice run.
+
+1. Commit the changes
+
+   ```shell
+   # Add the new files
+   git add .
+   # Commit the changes
+   git commit -m "My new files"
+   ```
+
+1. Push the changes to your forked repository
+
+   ```shell
+   git push origin feature_xyz
+   ```
+
+1. Create a pull request from your new branch
+
+   Go to the repository page on github.com. There, you will see a banner to create
+   the pull request.
+
+   ![pull request](images/pull-request-branch.png)
+
+1. Go to the pull request tab and celebrate!
+
+   ```shell
+    https://github.com/snowex-hackweek/learning-git/pulls
+    ```
+   
+1. Once your Pull Request is merged, clean up and delete the branch
+   locally and on the website.
```

Comment:
Will this be covered anywhere in the tutorial; the local and remote clean up?

Reply:
Good point. I can add another section regards that.